extensions/import/src/dialogs/derivedColumnDialog.ts

```diff
@@ -0,0 +1,370 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { ImportDataModel } from '../wizard/api/models';
+import * as EventEmitter from 'events';
+import { FlatFileProvider } from '../services/contracts';
+import * as constants from '../common/constants';
+
+const headerLeft: azdata.CssStyles = {
+	'border': 'none',
+	'text-align': 'left',
+	'white-space': 'nowrap',
+	'text-overflow': 'ellipsis',
+	'overflow': 'hidden',
+	'border-bottom': '2px solid'
+};
+
+const styleLeft: azdata.CssStyles = {
+	'border': 'none',
+	'text-align': 'left',
+	'white-space': 'nowrap',
+	'text-overflow': 'ellipsis',
+	'overflow': 'hidden',
+};
+
+export class DerivedColumnDialog {
+	private _dialogObject: azdata.window.Dialog;
+	private _doneEmitter: EventEmitter = new EventEmitter();
+	private currentTransformation: string[] = [];
```

Comment:
naming convention is inconsistent even in this class, no underscore prefix for private variables